lua/elixir/init.lua

```diff
@@ -77,7 +75,7 @@ local define_user_command = function()
     complete = function(_, cmd_line)
       local cmd = vim.trim(cmd_line)
       if vim.startswith(cmd, "Elixir nextls") then
-        return { "uninstall", "to-pipe", "from-pipe" }
+        return { "uninstall", "to-pipe", "from-pipe", "alias-refactor" }
```

Comment:
We should do the sales person thing here and maybe put uninstall last, or order them alphabetically if you want?

Reply:
Uninstall last, yes. Others doesn't matter as much